packages/mux-player/test/player.test.js

```diff
@@ -82,4 +82,50 @@ describe("<mux-player>", () => {
 
     assert.equal(String(Math.round(player.currentTime)), 3, "is about 3s in");
   });
+
+  it("video attributes are forwarded to media element", async function () {
+    this.timeout(10000);
+
+    const player = await fixture(`<mux-player
+      playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
+      muted
+    ></mux-player>`);
+
+    const muxVideo = player.video;
+
+    // controls should not be forwarded! player handles show/hide media-chrome.
+    player.setAttribute("controls", "");
+    assert(!muxVideo.hasAttribute("controls"), `has no controls attr added`);
+
+    const checkProps = {
+      autoplay: true,
+      muted: true,
+      playsinline: true,
+      loop: true,
+      crossOrigin: "",
+      preload: "metadata",
+      poster:
+        "https://image.mux.com/xLGf7y8cRquv7QXoDB02zEe6centwKfVmUOiPSY02JhCE/thumbnail.jpg?time=0",
+      src: "https://stream.mux.com/r4rOE02cc95tbe3I00302nlrHfT023Q3IedFJW029w018KxZA.m3u8",
+    };
+
+    for (let propName in checkProps) {
+      const attrName = propName.toLowerCase();
+      const attrValue = checkProps[attrName];
+
+      player.setAttribute(attrName, attrValue === true ? "" : attrValue);
+      assert(muxVideo.hasAttribute(attrName), `has ${attrName} attr added`);
+
+      if (!["playsinline"].includes(attrName)) {
+        assert.equal(
+          muxVideo[attrName],
+          attrValue,
+          `has true ${attrName} prop`
+        );
+      }
```

Comment:
instead of not checking `playsInline` should be check for that in the attrName?

```suggestion
      assert.equal(
        muxVideo[attrName === 'playsinline' ? 'playsInline' : attrName],
        attrValue,
        `has true ${attrName} prop`
      );
```

Reply:
it's not #66 actually, it's the issue @cjpillsbury brought up with removing attrs not having any effect.

but good call, maybe it's best to fix @66 in this PR too. this is about that mux-video is removing attributes when it shouldn't. only boolean attributes can be removed by changing the associated property to false. at least if we want to keep the setters / getters behave more like native elements.

Reply:
I'm going to do the playsInline a bit different because the `.toLowerCase()` is there. good suggestion!

Reply:
Ah, cool. Didn't quite get that this change was different from #66. But yeah, it seems related enough to fix it now.